kaldb/src/test/java/com/slack/kaldb/chunkManager/IndexingChunkManagerTest.java

```diff
@@ -316,24 +334,49 @@ public void testAddMessage() throws Exception {
 
   private void testChunkManagerSearch(
       ChunkManager<LogMessage> chunkManager,
+      List<String> chunkList,
       String searchString,
       int expectedHitCount,
       int totalSnapshots,
       int expectedSnapshotsWithReplicas) {
 
-    SearchQuery searchQuery =
-        new SearchQuery(
-            MessageUtil.TEST_DATASET_NAME,
-            searchString,
-            0,
-            com.slack.kaldb.testlib.TemporaryLogStoreAndSearcherRule.MAX_TIME,
-            10,
-            1000);
-    SearchResult<LogMessage> result = chunkManager.query(searchQuery, Duration.ofMillis(3000));
+    KaldbLocalQueryService<LogMessage> kaldbLocalQueryService =
+        new KaldbLocalQueryService<>(chunkManager, Duration.ofSeconds(3));
+    KaldbSearch.SearchRequest.Builder searchRequestBuilder = KaldbSearch.SearchRequest.newBuilder();
+    KaldbSearch.SearchResult response =
+        kaldbLocalQueryService.doSearch(
+            searchRequestBuilder
+                .setDataset(MessageUtil.TEST_DATASET_NAME)
+                .setQueryString(searchString)
+                .setStartTimeEpochMs(0)
+                .setEndTimeEpochMs(
```

Comment:
any specific reason to add one extra indirection to just using ` Long.MAX_VALUE` ?

Reply:
Replaced it with `Long.MAX_VALUE`. I think we used MAX_TIME to better represent what the value was.